w1r3/go/main.go

```diff
@@ -119,20 +119,38 @@ func main() {
 
 	ctx := context.Background()
 	enableProfiler(*projectID, *deployment, *profileVersion)
-	cleanupTracing := enableTracing(ctx, *tracingRate, *projectID)
+	cleanupTracing, err := enableTracing(ctx, *tracingRate, *projectID)
+	if err != nil {
+		log.Fatalf("Error enabling Cloud Profiler exporter %v", err)
+	}
 	defer cleanupTracing()
-	cleanupMeter := enableMeter(ctx, *projectID, instance.String())
+	cleanupMeter, err := enableMeter(ctx, *projectID, instance.String())
+	if err != nil {
+		log.Fatalf("Error enabling Cloud Trace exporter %v", err)
+	}
 	defer cleanupMeter()
 
-	transports := makeTransports(ctx, transportArgs)
+	transports, err := makeTransports(ctx, transportArgs)
+	if err != nil {
+		log.Fatalf("Error creating transports %v", err)
+	}
+	for _, t := range transports {
```

Comment:
This check can be skipped; there should never be a nil error and a nil client.

Reply:
Done.